controllers/chat/sendSignedMessage.js

```diff
@@ -0,0 +1,79 @@
+const Validator = require('fastest-validator');
+const {StreamChat} = require('stream-chat');
+const {responseError, responseSuccess} = require('../../utils/Responses');
+const {CHANNEL_TYPE} = require('../../helpers/constants');
+
+const v = new Validator();
+
+/**
+ *
+ * @param {import("express").Request} req
+ * @param {import("express").Response} res
+ */
+const sendSignedMesage = async (req, res) => {
+  const schema = {
+    channelId: 'string|empty:false',
+    message: 'string|empty:false',
+    channelType: 'number|empty:false'
+  };
+  const validated = v.validate(req.body, schema);
+  if (validated.length)
+    return res.status(403).json({
+      message: 'Error validation',
+      error: validated
+    });
+
+  const {channelId, message, channelType} = req.body;
+
+  let channelTypeDef;
+
+  switch (channelType) {
+    case CHANNEL_TYPE.CHAT:
+    case CHANNEL_TYPE.ANONYMOUS:
+      channelTypeDef = 'messaging';
+      break;
+
+    case CHANNEL_TYPE.GROUP:
+      channelTypeDef = 'group';
+      break;
+
+    default:
+      break;
+  }
+
+  if (!channelTypeDef)
+    return res.status(403).json({
+      message: 'Error validation',
+      error: 'Channel type not found'
+    });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

With the above suggested change, these lines are no longer needed.



<!-- This is an auto-generated comment by OSS CodeRabbit -->